### PR TITLE
add env variable to decide the srtm4 out dir

### PR DESCRIPTION
**Pull request by houssemtouansi**

### CODE REVIEW

File: srtm4.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 BIN = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'bin')
 GEOID = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'data')
-SRTM_DIR = os.path.join(os.path.expanduser('~'), '.srtm')
+SRTM_DIR = os.getenv('SRTM_DIR')
```
Comment (carlodef) [inline srtm4.py]: I'd suggest keeping the default `~/.srtm` cache directory if the env var is not set
```suggestion
SRTM_DIR = os.getenv('SRTM_DIR')
if SRTM_DIR is None:
    SRTM_DIR = os.path.join(os.path.expanduser('~'), '.srtm')
```

File: srtm4.py [hunk 2]
```diff
@@ -52,7 +52,11 @@ def get_srtm_tile(srtm_tile, out_dir):
     """
 
     # check if the tile is already there
-    out_dir = os.path.abspath(os.path.expanduser(out_dir))
+    if SRTM_DIR:
```
Comment (carlodef) [inline srtm4.py]: I think that it is misleading for the caller to change the function arguments inside the function. I'd prefer not to change `out_dir` and check the env var when defining the global variable `SRTM_DIR`

Comment (diabou) [reply srtm4.py]: @carlodef Sorry I misunderstood the code a bit :). @houssemtouansi will push a better fix

**Comment by houssemtouansi** [comment]
Thank you Carlo for the comments , I pushed another version.
You can review it

**Comment by diabou** [comment]
@carlodef are Houssem's last chages ok for you ?

**Pull request merged**